src/main.js

```diff
@@ -39,15 +41,9 @@ makeCoverButton.addEventListener('click', makeNew);
 createNewBookButton.addEventListener('click', makeMyBook);
 
 savedCoversSection.addEventListener('dblclick', function(event){
-  if(event.target.className === 'cover-image'){
-    var coverElement = event.target.parentElement;
-    var coverId = coverElement.id;
-    coverElement.parentNode.removeChild(coverElement);
-    for (var i = 0 ; i < savedCovers.length ; i ++) {
-      if (savedCovers[i].id === Number(coverId)) {
-        savedCovers.splice(i, 1)
-      }
-    }
```

Comment:
Should this bit of logic remain so that they underlying data model - `savedCovers` - reflects what is rendered in the DOM?

Reply:
Hmm yeah, that's weird. Maybe this was because the PR was approved and this PR is technically "revert". This should definitely be in there